sunflower-land/sunflower-land
# Feat/Goblin Mint/Bank Updates

# Description
This PR aims to add a narrative to the cooldown period that comes with minting a [rare item](https://docs.sunflower-land.com/crafting-guide/farming-and-gathering#crafting-limits). The story being that goblins are expert craftsman and need 7 days to complete the building of your rare item.

Note: I am working with assets that have not be finalised yet.

This PR also adds some UI updates for the bank modals. You can now move between withdrawing tokens and items easily. These modals need more work in the future but this is a good first step. 
<img width="380" alt="Screen Shot 2022-05-07 at 11 34 48 am" src="https://user-images.githubusercontent.com/17863697/167232853-e0d12dad-26ab-47a2-a57e-f0bc08a376a7.png">

<img width="380" alt="Screen Shot 2022-05-07 at 11 34 01 am" src="https://user-images.githubusercontent.com/17863697/167232861-81975bf2-e7b9-49be-9956-92a5a42d1e32.png">

<img width="380" alt="Screen Shot 2022-05-07 at 11 35 48 am" src="https://user-images.githubusercontent.com/17863697/167232869-ab392265-c674-462b-9a5e-c150f9a0fe9a.png">

<img width="380" alt="Screen Shot 2022-05-07 at 11 38 45 am" src="https://user-images.githubusercontent.com/17863697/167232902-5c525f5c-9cee-4d84-9925-2b32dbed65ea.png">


I have also added an `error` color to tailwind config.a


## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Go to Goblin Land
- Mint and item
- Confirm the cooldown period is visible. Does the story make sense?
- Check out the bank modals

# Checklist:
